apps/test-app/src/frontend/appui/frontstages/EditorFrontstage.tsx

```diff
@@ -12,12 +12,12 @@ import {
   StagePanelSection,
   StageUsage,
   StandardContentLayouts,
-  ToolbarItemUtilities,
-  ToolbarOrientation,
-  ToolbarUsage,
+  // ToolbarItemUtilities,
+  // ToolbarOrientation,
+  // ToolbarUsage,
   UiItemsProvider,
 } from "@itwin/appui-react";
-import { CreateArcTool, CreateLineStringTool } from "@itwin/editor-frontend";
+// import { CreateArcTool, CreateLineStringTool } from "@itwin/editor-frontend";
```

Comment:
Should this be removed?

Reply:
[Remove unused editor tools](https://github.com/iTwin/appui/pull/1291/commits/1f871cd55405658ef6340f73e54b408b95139516)